models/marketo__email_sends.sql

```diff
@@ -73,13 +73,13 @@ with sends as (
 
     select 
         booleans.*,
-        {% if var('marketo__enable_campaigns', False) %}
+        {% if var('marketo__enable_campaigns', True) %}
         campaigns.campaign_type,
         campaigns.program_id,
```

Comment:
Can you also update these [except clauses](https://github.com/fivetran/dbt_marketo/blob/a7827ee846768b9c2d0bf978c31b47e23613a81c/integration_tests/tests/consistency/consistency_email_sends.sql#L9) in the validation tests to exclude the `campaign_type` and `program_id` from the validation test if `marketo__enable_campaigns` is false. This way we can avoid false failures on validation tests in the future.

Reply:
Thanks Joe, I added a conditional to exclude `campaign_type` and `program_id` when `marketo__enable_campaigns` is set to False. Let me know what you think. To test, I set to `marketo__enable_campaigns` to false. Now `consistency_email_sends` passes. Keeping in mind this disables the `consistency_programs` and `consistency_campaigns` tests, but those tests are already addressed in our internal ticket thread.

![image](https://github.com/user-attachments/assets/5acc3f5b-495f-4f1c-b388-96b1e3af17b4)